### ISSUE TITLE
Removing ReLu activations brings the mAP of the final quantized model to zero!

### ISSUE THREAD

**Issue by ambitious-octopus** [ambitious-octopus/model_optimization#2]
We want to remove the [ReLU operations](https://github.com/sony/model_optimization/blob/8461a41fa8b6b57ce422fdd5f7301e9a9c8a1b20/tutorials/mct_model_garden/models_pytorch/yolov8/yolov8.py#L269-L272) in the Detect Head. However we got zero values for the mean Average Precision (mAP) after the quantization process if ReLU activations are removed. 
It would be optimal to be able to export the original model without any extra activation introduced. Is there a specific explanation for this? It would certainly be useful to understand this behavior.

**Comment by Idan-BenAmi** [comment]
Yes, there is a specific explanation for this, but first let me comment that due to some recent improvements we've add to our tools this modification will not be necessary.

The reason, in short, for this additional ReLU, was to bound the quantization range to be always positive (since we know the coordinates are positive). But as I mentioned, in the next MCT version, this modification will not be necessary.

Thanks
Idan
